squish.user.js

```diff
@@ -0,0 +1,291 @@
+// ==UserScript==
+// @name         At-a-glance GYR Hub client layout
+// @namespace    http://getyourrefund.org/
+// @version      0.1
+// @description  Rearrange the clients table to show more important info first.
+// @match        https://*.getyourrefund.org/en/hub
+// @match        https://*.getyourrefund.org/en/hub/clients
+// @match        https://*.getyourrefund.org/en/hub/clients?*
+// @match        https://*.getyourrefund.org/en/hub/clients/sla-breaches?*
+// @match        https://*.getyourrefund.org/en/hub/clients/sla-breaches
+// ==/UserScript==
+
+// // <script data-main="scripts/app" src="scripts/require.js"></script>
+// let require_script = document.createElement('script');
+// require_script.src = 'scripts/require.js';
+// document.body.appendChild( require_script );
+// let moment_script = document.createElement('<script src="moment.js"></script>');
+// let days_script = document.createElement('<script src="moment-business-days.js"></script>');
+
+// let business_days = require('moment-business-days');
+
+var squish = function () {
+
+  // Prevent stuff from being added multiple times (implement after development)
+  if ( document.querySelector( '.squish' )  ) {
+    console.log( 'squish has already run' );
+    return;
+  }
+
+  document.body.className += ' squish';
+
+  // // Be able to turn dates into business days
+  // let moment_script = document.createElement('script');
+  // moment_script.src = 'moment.js';
+  // document.body.appendChild( moment_script );
+  // let days_script = document.createElement('script');
+  // days_script.src = 'moment-business-days.js';
+  // document.body.appendChild( days_script );
+
+  // let diff = moment('Mar 12 2:13 AM', 'MMMM DD LT').businessDiff(moment());
+  // console.log( diff );
+
+  // Turning the dates into business days
+  // IMPORTANT: Only works for client dates in the same year as today. GYR column
+  // excludes the year, so I use `today` to add today's year to whatever data is
+  // present in the relevant column.
+  let today = new Date();
+  let this_year = today.getFullYear();
+  let one_day_ms = 1000 * 60 * 60 * 24;
+
+  let column_headers = document.querySelectorAll('table.client-table thead th');
+  let name_header_index = 1;
+  let id_index = 2;
+  let org_header_index = 3;
+  let lang_header_index = 4;
+  let unemployment_index = 5;
+  let updated_index = 6;
+  let waiting_index = 7;
+  let created_index = 8;
+  for ( let header_i = 0; header_i < column_headers.length; header_i++ ) {
+    let text = column_headers[ header_i ].innerText;
```

Comment:
Should probably add
```suggestion
    let text = column_headers[ header_i ].innerText.toLowerCase().trim();
```

Reply:
Can't commit this as the logic would have to be changed as well, but good suggestion.